src/Tests/Tests/Aggregations/Bucket/GeoHashTile/GeoTileGridAggregationUsageTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using FluentAssertions;
+using Nest;
+using Tests.Core.Extensions;
+using Tests.Core.ManagedElasticsearch.Clusters;
+using Tests.Domain;
+using Tests.Framework.EndpointTests.TestState;
+using static Nest.Infer;
+
+namespace Tests.Aggregations.Bucket.GeoTileGrid
+{
+	public class GeoTileTileAggregationUsageTests : AggregationUsageTestBase
+	{
+		public GeoTileTileAggregationUsageTests(ReadOnlyCluster i, EndpointUsage usage) : base(i, usage) { }
+
+		protected override object AggregationJson => new
+		{
+			my_geotile = new
+			{
+				geotile_grid = new
+				{
+					field = "locationPoint",
+					precision = 3,
+					size = 1000,
+					shard_size = 100
+				}
+			}
+		};
+
+		protected override Func<AggregationContainerDescriptor<Project>, IAggregationContainer> FluentAggs => a => a
+			.GeoTile("my_geotile", g => g
+				.Field(p => p.LocationPoint)
+				.Precision(GeoTilePrecision.Precision3)
+				.Size(1000)
+				.ShardSize(100)
+			);
+
+		protected override AggregationDictionary InitializerAggs =>
+			new GeoTileGridAggregation("my_geotile")
+			{
+				Field = Field<Project>(p => p.LocationPoint),
+				Precision = GeoTilePrecision.Precision3,
+				Size = 1000,
+				ShardSize = 100
+			};
+
+		protected override void ExpectResponse(ISearchResponse<Project> response)
+		{
+			response.ShouldBeValid();
+			var myGeoTileTile = response.Aggregations.GeoTile("my_geotile");
+			myGeoTileTile.Should().NotBeNull();
```

Comment:
Can we assert some of the buckets on `myGeoTileTile` to be a ta more rigid?

Reply:
Added some additional assertions.